Make core_aten_decomp to be alias to export table

tugsbayasgalan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #140086

Differential Revision: [D64554098](https://our.internmc.facebook.com/intern/diff/D64554098/)

tugsbayasgalan committed:
Make core_aten_decomp to be alias to export table



Differential Revision: [D64554098](https://our.internmc.facebook.com/intern/diff/D64554098/)

[ghstack-poisoned]

bdhirsh commented on torch/_decomp/__init__.py:
```diff
@@ -262,25 +266,16 @@ def remove_decompositions(
-    return decomp_table
-
-
 def _core_aten_decompositions_post_autograd() -> (
```
I guess at some point later we could delete this code (and some of the other code in this file) that needed to sweat a bunch to generate the old core aten decomps?

tugsbayasgalan committed:
Update on "Make core_aten_decomp to be alias to export table"


Differential Revision: [D64554098](https://our.internmc.facebook.com/intern/diff/D64554098/)

[ghstack-poisoned]

tugsbayasgalan commented:
@pytorchbot rebase

tugsbayasgalan committed:
Update

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Make core_aten_decomp to be alias to export table"


Differential Revision: [D64554098](https://our.internmc.facebook.com/intern/diff/D64554098/)

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D64554098).
